src/src/CustomBuild/ESPEasyDefaults.h

```diff
@@ -385,6 +385,15 @@
 #endif
 
 
+// --- We define the default features to be enabled here if custom.h is not used
+#ifndef USE_CUSTOM_H
```

Comment:
This is not the correct check.
`USE_CUSTOM_H` is only defined if there is a Custom.h file found.
The correct check should be `PLUGIN_BUILD_CUSTOM`

Reply:
because is a #ifndef means that for all the other builds that do not use custom.h this feature is enabled.
So it does not currently affects any of the existing builds.

Reply:
> because is a #ifndef means that for all the other builds that do not use custom.h this feature is enabled. So it does not currently affects any of the existing builds.

I know, but when you create a custom build using the Python scripts, instead of the Custom.h, you have a different behavior compared to when using Custom.h

Reply:
if I change the USE_CUSTOM_H to PLUGIN_BUILD_CUSTOM all the other builds will not have this feature enabled.
This will break all the existing builds that rely on this feature.

Reply:
ok. I changed it to PLUGIN_BUILD_CUSTOM